corehq/ex-submodules/pillowtop/pillow/interface.py

```diff
@@ -264,7 +265,8 @@ def fire_change_processed_event(self, change, context):
 def handle_pillow_error(pillow, change, exception):
     from pillow_retry.models import PillowError
     error_id = None
-    if pillow.retry_errors:
+    # always retry document missing errors, because the error is likely with couch
+    if pillow.retry_errors or isinstance(exception, DocumentMissingError):
```

Comment:
Is it possible to set a limit to this? 
Considering that if a document is actually missing (if even possible for such pillow processes) it would just keep retrying it?

Reply:
Just meant that if a document is actually missing and not because of the situation we have here, this doc that needs to be processed will not be endlessly attempted.

Reply:
that's true but also very unlikely since the change feed comes from couchdb